ui-v2/app/styles/components/icons/index.scss

```diff
@@ -79,6 +80,15 @@
   margin-top: -10px;
   background-color: $color-transparent;
 }
+%with-proxy::before {
+  @extend %pseudo-icon;
+  background-image: url('data:image/svg+xml;charset=UTF-8,<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg"><g fill-rule="nonzero" fill="none"><path d="M2.242 7.138l6.963.774a1 1 0 0 1 .883.883l.774 6.963a1 1 0 0 1-1.104 1.104l-6.963-.774a1 1 0 0 1-.883-.883l-.774-6.963a1 1 0 0 1 1.104-1.104z" stroke="%23BAC1CC" fill="%23FAFBFC"/><path d="M5.242 4.138l6.963.774a1 1 0 0 1 .883.883l.774 6.963a1 1 0 0 1-1.104 1.104l-6.963-.774a1 1 0 0 1-.883-.883l-.774-6.963a1 1 0 0 1 1.104-1.104z" stroke="%238E96A3" fill="%238E96A3"/><path d="M8.242 1.138l6.963.774a1 1 0 0 1 .883.883l.774 6.963a1 1 0 0 1-1.104 1.104l-6.963-.774a1 1 0 0 1-.883-.883l-.774-6.963a1 1 0 0 1 1.104-1.104z" stroke="%23BAC1CC" fill="%23FAFBFC"/></g></svg>');
```

Comment:
This works fine, but it's a little gnarly having to account for the `#` encoding whenever adding an svg. Other UIs use [ember-inline-svg](https://github.com/minutebase/ember-inline-svg) to help treat svgs like svgs. [ember-svg-jar](https://github.com/ivanvotti/ember-svg-jar) is also popular.

Granted these are slightly different, since they put the svg in the DOM, but that also makes it possible to style the svg dynamically.

Reply:
Those are good options. If we were inlining SVGs into HTML I'd definitely use one of those, but given we're using inline CSS we can’t use them here.